src/domain/registration/form/partial/ChildFormField.tsx

```diff
@@ -1,46 +1,57 @@
-import * as React from 'react';
-import { ArrayHelpers } from 'formik';
+import React from 'react';
+import { ArrayHelpers, Field, FormikState, FormikHelpers, getIn } from 'formik';
 import { useTranslation } from 'react-i18next';
+import { Button, TextInput } from 'hds-react';
 
 import styles from './childFormField.module.scss';
 import { formatTime, newMoment } from '../../../../common/time/utils';
 import { DEFAULT_DATE_FORMAT } from '../../../../common/time/TimeConstants';
-import EnhancedInputField from '../../../../common/components/form/fields/input/EnhancedInputField';
-import InputField from '../../../../common/components/form/fields/input/InputField';
-import SelectField from '../../../../common/components/form/fields/select/SelectField';
 import { Child } from '../../../child/types/ChildTypes';
 import { getTranslatedRelationshipOptions } from '../../../child/ChildUtils';
 import Icon from '../../../../common/components/icon/Icon';
 import happyChildIcon from '../../../../assets/icons/svg/childFaceHappy.svg';
 import deleteIcon from '../../../../assets/icons/svg/delete.svg';
-import Button from '../../../../common/components/button/Button';
 import { validatePostalCode } from '../../../../common/components/form/validationUtils';
+import FormikDropdown, {
+  HdsOptionType,
+} from '../../../../common/components/form/fields/dropdown/FormikDropdown';
+import { RegistrationFormValues } from '../../types/RegistrationTypes';
 
-interface ChildFormFieldProps {
+type ChildFormFieldProps = {
   child: Child;
   childIndex: number;
   arrayHelpers: ArrayHelpers;
-}
+} & Pick<FormikState<RegistrationFormValues>, 'errors' | 'touched'> &
+  Pick<
+    FormikHelpers<RegistrationFormValues>,
+    'setFieldTouched' | 'setFieldValue'
+  >;
```

Comment:
Could this API become clearer if you just accepted the formik objects as is?

Etc.
```
formikState: FormikState<RegistrationFormValues>,
formikHelpers: FormikHelpers<RegistrationFormValues>,
```

Reply:
Probably. But that requires refactoring of the registration form, I would like to save that for later.

Reply:
So resolving it